Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3
+
+ARG BUILD_ENV
+
+ENV BUILD_ENV=${BUILD_ENV}
+
+RUN apt update && \ 
+	apt install -y python3 graphviz
+	
+RUN pip install poetry
+
+WORKDIR /usr/src/app
+
+COPY poetry.lock pyproject.toml ./
+
+RUN poetry config virtualenvs.create false \
+  && poetry install --no-interaction --no-ansi --only main
+
+# TODO move this after poetry install but then pyprojet.toml packages entry cannot find wireviz_web
+COPY . .
+  
+ENV FLASK_APP=wireviz_web
+
+EXPOSE 3005 5000

```

Comment:
What's on port 5000? Can you remove this port otherwise?

Reply:
It was from some testing during the Dockerfile construction, it is the default Flask port, I will remove it.